components/export/api/src/main/java/org/phenotips/export/internal/SheetAssembler.java

```diff
@@ -212,8 +215,8 @@ private DataSection assembleSectionsY(List<DataSection> sections, Boolean finali
     }
 
     /**
-     * @return a {@link org.phenotips.export.internal.DataSection} that contains all {@link
-     * org.phenotips.export.internal.DataCell}s
+     * @return a {@link org.phenotips.export.internal.DataSection} that contains all
+     *         {@link org.phenotips.export.internal.DataCell}s
```

Comment:
The proper syntax is the changed one. However, Anton is right, cosmetic changes shouldn't be included along with code in the same pull request, but in their own pull request. Otherwise, it's harder to review.